src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -87,10 +89,17 @@ function presentTransaction (transaction) {
   const { state, origin, event, timestamp, adjustment } = transaction
   const stateStage = state.split('/')[1]
   const stateDirection = state.split('/')[0] // NOTE: This returns either 'incoming' or 'outgoing,' wherein, 'incoming' indicates the recipient of funds, 'outgoing' indicates the spender of funds.
+  // NOTE: *Holofuel does NOT yet provide a balance that represents the 'RESULTING ACCT BALANCE after this transaction adjustment', instead of the only the tx adjustment balance or real-time balance.*
+  const adjustmentValues = Object.values(adjustment).map(key => key.Ok)
+  const adjustmentKeys = Object.keys(adjustment)
+  const parsedAdjustment = adjustmentValues.reduce((result, field, index) => {
```

Comment:
you can use `lodash.mapValues here`

something like 
`import { mapValues} from 'lodash/fp'` <-- note the fp

`const parsedAdjustment = mapValues('Ok', object)`

or if that doesn't work

`const parsedAdjustment = mapValues(key => key.Ok, object)`